crates/rolldown/src/module_loader/module_loader.rs

```diff
@@ -402,35 +402,37 @@ impl ModuleLoader {
         acc
       },
     );
+
     self.shared_context.plugin_driver.set_context_load_modules_tx(None).await;
 
     let modules: IndexVec<ModuleIdx, Module> = self
       .intermediate_normal_modules
       .modules
       .into_iter()
-      .flatten()
       .enumerate()
-      .map(|(id, mut module)| {
-        let id = ModuleIdx::from(id);
-        if let Some(module) = module.as_normal_mut() {
-          // Note: (Compat to rollup)
-          // The `dynamic_importers/importers` should be added after `module_parsed` hook.
-          let importers = std::mem::take(&mut self.intermediate_normal_modules.importers[id]);
-          for importer in &importers {
-            if importer.kind.is_static() {
-              module.importers.push(importer.importer_path.clone());
-            } else {
-              module.dynamic_importers.push(importer.importer_path.clone());
+      .filter_map(|(id, module)| {
```

Comment:
I don't think `filter_map` is proper too. Should just use `unwrap` for such case. Having `None` in the `intermediate_normal_modules` is invalid and shouldn't be ignored.

Reply:
Got it.